website/src/blogs/2018-04-13-how-do-i-create-an-openebs-storage-pool-on-google-persistent-disk.md

```diff
@@ -0,0 +1,145 @@
+---
+title: How do I create an OpenEBS storage pool on Google Persistent Disk
+author: Karthik Satchitanand
+author_info: Karthik has been into the Design and Development of tools for infrastructure as code, software testing performance & benchmarking & chaos engineering.
+date: 13-04-2018
+tags: Docker, OpenEBS, Solutions, Google Cloud Platform, Kubernetes
+excerpt: The OpenEBS volume replicas, which are the actual backend storage units of the OpenEBS iSCSI target currently store the data in a hostPath on the Kubernetes nodes.
+---
+
+This article belongs to #HowDoI series on Kubernetes and OpenEBS.
+
+The OpenEBS volume replicas, which are the actual backend storage units of the OpenEBS iSCSI target currently store the data in a hostPath on the Kubernetes nodes. By default, a folder with the volume (PV) name is created on the root filesystem, in a parent directory (/var/openebs) & bind mounted into the container during the replica pod instantiation. This parent directory (also created if not already available), which is basically a persistent path to hold the individual volumes is referred to as a **_Storage Pool_**.
+
+Note: The notion of the storage pool described above is specific to the current default storage engine ,i.e., Jiva. Future releases may see availability of additional storage-engines which can consume block devices instead of hostdir to create storage pools
```

Comment:
```suggestion
Note: The notion of the storage pool described above is specific to the current default storage engine,i.e., Jiva. Future releases may see availability of additional storage-engines which can consume block devices instead of hostdir to create storage pools
```